config/crd/bases/solr.apache.org_solrbackups.yaml

```diff
@@ -71,7 +71,7 @@ spec:
                 description: The location to store the backup in the specified backup repository.
                 type: string
               persistence:
-                description: Persistence is the specification on how to persist the backup data.
+                description: 'Persistence is the specification on how to persist the backup data. This feature has been removed as of v0.5.0. Any options specified here will not be used. TODO: Remove this field entirely in v0.6.0'
```

Comment:
[0] It's a little odd to see a TODO comment in what (I think?) is user-facing documentation.  Should this be in a `#`-based YAML comment instead?

Reply:
Fair enough on removing the TODO, because we might want to leave in the field anyways (for yaml back-compat).

These files are auto-generated by `make manifests` after you edit anything in the `api/` package. So we can't really add yaml comments here. These descriptions are taken from the comments before the property in the struct.